### ISSUE TITLE
HTML examples should be valid html

### ISSUE THREAD

**Issue by erictheise** [platform-coop-toolkit/pinecone#287]
In the case of Checkboxes and Radio Buttons the example code in the HTML section displays multiple `input`s having the same `id`

```
<ul class="input-group">
    <li><input id="radio" type="radio" name="radio" value="radio" /><label for="radio">Radio Button</label></li>
    <li><input id="radio" type="radio" name="radio" value="radio" checked /><label for="radio">Radio Button</label></li>
    <li><input id="radio" type="radio" name="radio" value="radio" disabled /><label for="radio">Radio Button</label></li>
</ul>
```

and so the examples don't actually work. Clicking the second checkbox toggles the first. It is impossible to select the second radio button once the first has been selected.

I haven't looked behind the scenes to see how the examples are generated but they should contain unique indices and exhibit correct behavior for the `input` type.

**Comment by greatislander** [comment]
Should be resolved in #240.

**Comment by erictheise** [comment]
👍
